drivers/sensors/fakesensor_uorb.c

```diff
@@ -197,6 +197,20 @@ static inline void fakesensor_read_gyro(FAR struct fakesensor_s *sensor,
                     sizeof(struct sensor_gyro));
 }
 
+static inline void fakesensor_read_baro(FAR struct fakesensor_s *sensor,
+                                        uint64_t event_timestamp)
+{
+  struct sensor_baro baro;
+  char raw[50];
```

Comment:
Suggestion, replace hard-coded 50 with some #define BARO_SAMPLES_NUM 50, this way people could find and modify it easily.

Reply:
I could add this, but it might be worth adding such configuration options to the rest of the fakesensors as well for consistency at that point (which all use this 50 character hardcoded length). I think it's probably fine to just leave for now since it would be hard to get past 50 chars using two floats.